routes/clients.js

```diff
@@ -195,6 +195,43 @@ router.delete('/:idClient/products/:idProduct/favorites', (req, res) => {
             res.sendStatus(200)
         }
     })
+});
+
+/////////////////////////////////////////////////////// Favorites shops //////////////////////////////////////////////////////////
+
+//GET all the favorite shops of one client by it's ID
+router.get('/:idClient/favorites/shops', (req, res) => {
+    connection.query('SELECT * FROM favorite_shop AS f JOIN shop AS s ON f.id_shop = s.id WHERE id_client = ?', req.params.idClient, (err, result) => {
```

Comment:
Pour une meilleur visibilité aussi, je pense que créer une constante idClient.
```suggestion
router.get('/:idClient/favorites/shops', (req, res) => {
    const IdClient = req.params.idClient;
    connection.query('SELECT * FROM favorite_shop AS f JOIN shop AS s ON f.id_shop = s.id WHERE id_client = ?', idClient, (err, result) => {
```